python/lsst/summit/utils/consdbClient.py

```diff
@@ -485,4 +485,4 @@ def schema(self, instrument: str, table: str) -> dict[str, tuple[str, str]]:
         """
         url = _urljoin(self.url, "schema", quote(instrument), quote(table))
         result = self._handle_get(url)
-        return {key: tuple(value) for key, value in result.items()}
+        return {key: (str(value[0]), str(value[1])) for key, value in result.items()}
```

Comment:
I think you need to check with @ktlim that these are actually strings and that cast is OK.